test/test_flake8.py

```diff
@@ -14,50 +14,27 @@
 
 from __future__ import print_function
 
-import os
+import pathlib
+import subprocess
 import sys
 
-from flake8.api.legacy import get_style_guide
 import pytest
 
 
 @pytest.mark.flake8
 @pytest.mark.linter
 def test_flake8():
-    style_guide = get_style_guide(
-        exclude=['conf.py'],
-        ignore=[
-            'C402',  # ignore presence of unnecessary generators
-            'C405',  # ignore presence of unnecessary literals
-            'C407',  # ignore presence of unnecessary comprehensions
-            'C408',  # ignore presence of unnecessary tuple/list/dict
-            'D',  # ignore documentation related warnings
-            'F401',  # ignore presence of unused imports
-            'F841',  # ignore presence of unused variables
-            'I',  # ignore import order related warnings
-            'N802',  # ignore presence of upper case in function names
-            'W504',  # ignore line breaks after binary operator (new rule added in 2018)
-        ],
-        max_line_length=200,
-        max_complexity=10,
-        show_source=True,
+    # flake8 doesn't have a stable public API as of ver 6.1.0.
+    # See: https://flake8.pycqa.org/en/latest/user/python-api.html
+    # Calling through subprocess is the most stable way to run it.
+    result = subprocess.run(
+        ['flake8'],
+        stdout=subprocess.PIPE,  # capture_output doesn't work on py3.6
+        stderr=subprocess.PIPE,
```

Comment:
Is there any reason to capture the output? Maybe we should just let it bleed through to the parent process.

```suggestion
        stdout=subprocess.PIPE,  # capture_output doesn't work on py3.6
        stderr=subprocess.PIPE,
```

Reply:
Additionally, `subprocess.run` isn't available in Python 2, so we can use `subprocess.call` instead, which returns the process exit code instead of a CompletedProcess (since we're not capturing the output anymore, we don't really care).

Reply:
> Is there any reason to capture the output? Maybe we should just let it bleed through ...

It was a long while ago, so I forgot why. I tested and there wasn't any difference, I think we can delete this capture.

> Python 2 support

Will do.